vaadinfrontend/src/main/resources/application.properties

```
@@ -11,7 +12,7 @@ spring.datasource.url=${USER_DB_URL:localhost}
 spring.datasource.driver-class-name=${USER_DB_DRIVER:com.mysql.cj.jdbc.Driver}
 spring.datasource.username=${USER_DB_USER_NAME:myusername}
 spring.datasource.password=${USER_DB_USER_PW:astrongpassphrase!}
-spring.jpa.hibernate.ddl-auto=none
+spring.jpa.hibernate.ddl-auto=update
```

Do we really want to allow auto update of hibernate permanently? This could lead to cluttering of the database during development

---

No this was only for developing.
```suggestion
spring.jpa.hibernate.ddl-auto=none
```